packager/media/formats/mp4/mp4_muxer.cc

```diff
@@ -269,7 +269,8 @@ Status MP4Muxer::DelayInitializeMuxer() {
 
     // Generate EditList if needed. See UpdateEditListOffsetFromSample() for
     // more information.
-    if (edit_list_offset_.value() > 0) {
+    const bool isVideoStream = stream->stream_type() == kStreamVideo;
+    if (edit_list_offset_.value() > 0 && !isVideoStream) {
```

Comment:
We cannot just skip adding the EditList here as it will introduce problems elsewhere. Did you find the clause that EditList shall not be present in the spec?